### PR TITLE
Init menus

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: navbar.html [hunk 1]
```diff
@@ -42,7 +42,11 @@
         </div>
         <div br-slot="br-navbar-toolbar">
           <div class="navbar-right">
-            <div br-slot="br-navbar-tool"></div>
+            <div class="btn-toolbar">
```
Comment (dlongley) [inline navbar.html]: Does this force tools to be buttons? Perhaps we want the toolbar to have  a slot for tools and a built-in button toolbar with a different slot for each toolbar-button?

Comment (mattcollier) [inline navbar.html]: Here is the css for the btn-toolbar class: https://github.com/digitalbazaar/bedrock-angular/blob/master/css/app.css#L3329

Comment (dlongley) [inline navbar.html]: It's more about the semantics of it. Bootstrap probably expects only buttons to be in a btn-toolbar. That tells me that we should have two slots here ... one for entirely custom navbar-tools (br-navbar-tool) and then a built-in, by default empty, button toolbar that has a "br-navbar-toolbar-button" slot.

**Comment by mattcollier** [comment]
@dlongley recommendations implemented and tested with the login button and the user profile info in use in bedrock-idp.

**Pull request merged**